tapchannel/aux_funding_controller.go

```diff
@@ -1351,6 +1351,19 @@ func (f *FundingController) processFundingMsg(ctx context.Context,
 
 	tempPID = assetFunding.pid
 
+	// Whatever the message from the peer is, it's about funding a channel.
+	// We can only support asset channels if we have the correct proof
+	// courier type configured, so we're ready to receive the channel funds
+	// once the channel is (force) closed. This only works if we have a
+	// universe based proof courier configured. A hashmail based courier
+	// can't deal with the OP_TRUE funding output script key, as that's the
+	// same for asset channels out there. So the single mailbox would always
+	// be occupied.
+	if err := f.validateLocalProofCourier(); err != nil {
+		return tempPID, fmt.Errorf("unable to accept channel funding "+
+			"request, local proof courier is unsupported: %w", err)
```

Comment:
`local proof courier is unsupported` -> `local proof courier is invalid`

to correspond with the the general `validateLocalProofCourier` call. Also, i think that the comment here re universe/hashmail can just be in the body of method `validateLocalProofCourier`.

Same below in `processFundingReq`.